example/sofast_fixed/example_calculate_dot_locations_from_display_shape_file.py

```diff
@@ -0,0 +1,58 @@
+from os.path import join, dirname
+
+from opencsp.app.sofast.lib.DotLocationsFixedPattern import DotLocationsFixedPattern
+from opencsp.app.sofast.lib.DisplayShape import DisplayShape
+from opencsp.app.sofast.lib.SystemSofastFixed import SystemSofastFixed
+from opencsp.common.lib.deflectometry.ImageProjection import ImageProjection
+from opencsp.common.lib.opencsp_path.opencsp_root_path import opencsp_code_dir
+import opencsp.common.lib.tool.file_tools as ft
+import opencsp.common.lib.tool.log_tools as lt
+
+
+def example_calculate_dot_locations_from_display_shape():
```

Comment:
This name is ok, but may I instead suggest `example_create_dot_pattern_from_display_shape`?

Reply:
Yeah, that name is probably better, changed!